### PR TITLE
updated install electron instruction

### PR DESCRIPTION
**Pull request by homdx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ for dependencies required for building packages for various platforms.
 The only platform that can build packages for all three platforms is macOS:
 ```bash
 brew install mono
-yarn install
+yarn add install
```
Comment (t3chguy) [inline README.md]: this is not right, you're not trying to install a package called `install`

**Comment by jryans** [comment]
Thanks for making this contribution a while back. @homdx, do you plan to keep working on this?

**Comment by homdx** [comment]
@jryans @t3chguy updated pull request

File: README.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ To build it yourself, follow the instructions below.
 2. Install Electron and run it:
 
    ```bash
-   yarn electron
+   yarn add electron
```
Comment (t3chguy) [inline README.md]: if anything this should be 

```
yarn global add electron
yarn electron
```

Comment (t3chguy) [reply README.md]: You change it from `run it` to `Install electron`

**Comment by t3chguy** [comment]
Thanks for your contribution but this seems abandoned and/or confused, if you have any queries about the electron usage flows feel free to ask in https://matrix.to/#/#riot-web:matrix.org

**Pull request closed without merging**